.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings and Time Notice
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new issues and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvence this causes.'
+        pr-message: 'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new pull requests and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvence this causes.'
```

Comment:
```suggestion
        pr-message:  'Thank you for contributing and using this repository while learning GitHub. We have fallen behind on new issues and are resolving them in the order they were submitted. It maybe a number of weeks before we get to this one. Sorry for any inconvenience this causes. Choo choo! 🚂🌈❤️'
```

Reply:
The pr-message is for new pull requests. Did you mean to change it to say issue? I think pull request would make more sense would resolve be the best working for pull requests. Maybe we should go with accepting or merging.

Reply:
Yes, that was my mistake, thanks for catching!

Reply:
Fixed and used the term "reviewing" which I think works well here?